appveyor.yml

```diff
@@ -18,8 +18,7 @@ test_script:
   # Output useful info for debugging.
   - node --version
   - npm --version
-  # run tests
-  - npm test
+  - npm run test:ci
```

Comment:
I think there's no way this can run, because the actual install doesn't happen until `after_test:` (that was probably wrong for a while).

Maybe it should be renamed to `before_build`?

https://www.appveyor.com/docs/build-configuration/

Reply:
Tried to put it in the install, but got error [`was compiled against a different Node.js version using NODE_MODULE_VERSION 64. This version of Node.js requires NODE_MODULE_VERSION 59.`](https://ci.appveyor.com/project/modulesio/exokit/build/0.0.1898).

Trying a bunch of random things, lemme know if you have an idea the proper way.

Reply:
That one in particular seems because of `nodejs_version: "9"`: https://github.com/webmixedreality/exokit/pull/153/files#diff-180360612c6b8c4ed830919bbb4dd459R5

Reply:
(We need `10` to both build and run after building)